src/test/codegen/force-no-unwind-tables.rs

```diff
@@ -1,7 +1,9 @@
-// compile-flags: -C no-prepopulate-passes -C panic=abort -C force-unwind-tables=n
+// compile-flags: -C no-prepopulate-passes -C force-unwind-tables=n
 // ignore-windows
 
 #![crate_type="lib"]
 
 // CHECK-NOT: attributes #{{.*}} uwtable
```

Comment:
Similarly this needs to verify that these attributes are actually attached to the `foo` function. 

Something like what `src/test/codegen/ffi-pure.rs` does, perhaps.